src/assets.ts

```diff
@@ -161,12 +161,8 @@ export class AssetGenerator {
             throw new Error("Startup project not set");
         }
 
-        const startupProjectDir = path.dirname(this.startupProject.Path);
-        const relativeProjectDir = path.join('${workspaceFolder}', path.relative(this.workspaceFolder.uri.fsPath, startupProjectDir));
-        const configurationName = 'Debug';
```

Comment:
Not sure how MSBuild determines what the current Configuration is...but also, Debug might not exist since that's user-editable. So might be best to just stick with TargetPath because that's guaranteed to exist.

Reply:
The concern that I have is that this is generating the tasks and launch json files and they need to be in sync such that we will be launching the app that was built by the tasks.json build task. 

We aren't passing a specific configuration argument in the build task when building prior to launch or in O# when loading projects, so this might be a safe assumption, but would like a second opinion. @gregg-miskelly what do you think?

Reply:
I am assuming that neither OmniSharp or the default tasks.json task will specify a configuration. So it will use whatever the msbuild project/props/targets files decide is the default. It seems like in most cases the build and OmniSharp will wind up coming to the same default. I am sure there is some case where they could work out something different, but its not like the old code was exactly bullet proof -- it would have come up with the wrong answer any time that the user overwrote nearly any msbuild property.